programs/server/CMakeLists.txt

```diff
@@ -31,3 +31,4 @@ if (GLIBC_COMPATIBILITY)
 endif ()
 
 install(FILES config.xml users.xml DESTINATION ${CLICKHOUSE_ETC_DIR}/clickhouse-server COMPONENT clickhouse)
+install(FILES config.d/query_masking_rules.xml DESTINATION ${CLICKHOUSE_ETC_DIR}/clickhouse-server/config.d COMPONENT clickhouse)
```

Comment:
Looks like this file (or file with the same name) is already installed and symlinked to config.d folder.

That's the reason why fast-test is failing.

Reply:
My impression is that testing tools should provide workarounds for issues like this, not production builds. BTW, it should be fixed now in #13382